pkg/console/starter/starter.go

```diff
@@ -215,7 +230,9 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		informer.Start(ctx.Done())
 	}
 
+	go consoleServiceController.Run(1, ctx.Done())
 	go consoleOperator.Run(ctx.Done())
+
```

Comment:
nit: lets remove this one :)

Reply:
Will do.
I left the space between "our controllers" and "lib go controllers" but doesn't really do us much.